.github/actions/metabase/action.yaml

```diff
@@ -0,0 +1,39 @@
+name: Deploy Metabase to OpenShift
+description: Metabase action
+inputs:
+  openshift_server_url:
+    description: "URL of the OpenShift server"
+    required: true
+  openshift_token:
+    description: "Unique login token for OpenShift"
+    required: true
+  openshift_metabase_namespace:
+    description: "The namespace to be deployed to"
+    required: true
+  host_route:
+    description: "The route for metabase to be located at"
+    required: true
+  db_route:
+    description: "Location of the CrunchyDB DB"
+    required: true
+  db_password:
+    description: "Database password"
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Authenticate, set context, and run deploy script
+      uses: redhat-actions/oc-login@v1
+      with:
+        openshift_server_url: ${{ inputs.openshift_server_url }}
+        openshift_token: ${{ inputs.openshift_token }}
+        openshift_metabase_namespace: ${{ inputs.openshift_metabase_namespace }}
```

Comment:
```suggestion
```
The oc login doesn't need (and doesn't handle) the `openshift_metabase_namespace` env variable